src/pages/home/Home.jsx

```diff
@@ -0,0 +1,68 @@
+import { Link } from "react-router-dom";
+import { hero } from "../../assets";
+import "./home.css";
+import { CategoryCard, ProductCard } from "../../components";
+import { useProducts } from "../../context/product-context";
+import { useEffect } from "react";
+import axios from "axios";
+
+export function Home(){
+    const {state, dispatch} = useProducts();
+
+    /**
+     * useEffect to fetch products using API
+     */
+    useEffect(()=>{
+        (async function asyncProductFetch(){
+            try{
+                const {data} = await axios.get("/api/products");
```

Comment:
If you are using ES6 syntax, use it everywhere. You can change the async call to an arrow function as well.

Reply:
Yes, I can use ES6 but this looks more readable to me with function name. Inside useEffect I have used ES6 because it's an anonymous function.